traitsui/ui.py

```diff
@@ -352,7 +352,7 @@ def find(self, include):
 
         # Get the context 'object' (if available):
         if len(context) == 1:
-            object = list(context.values())[0]
+            object = next(context.values())
```

Comment:
This looks wrong to me; does this work? I'm assuming that `context` is a `dict`, in which case `context.values()` isn't an iterator in either Python 2 or Python 3. `object, = context.values()` would work, if you can put up with that kind of "grit-on-Tim's-monitor" syntax. Or just leave the original.

(Same comment again below for `next(context.keys())`.)

Reply:
`[object] = context.values()` is Yet Another Way. I'm told that some people like it.